src/css/tui-editor-contents.css

```diff
@@ -221,13 +221,22 @@
     list-style: none;
     padding-left: 22px;
     margin-left: -22px;
+}
+
+.tui-editor-contents ul li.task-list-item:before {
     background-repeat: no-repeat;
     background-size: 16px 16px;
-    background-position: 0 2px;
+    background-position: 50%;
```

Comment:
Because of this css, the task check box is located a little below the text. So I think this css do not need.

Reply:
It's needed because the background image is 16x16 pixels while the element is 18x18 pixels so it centers the image perfectly. I can just change the `top: 2px` to align it with the text ?

Reply:
I miss there is `top: 2px`. If use `background-position: 50%`, `top: 2px` is not needed to align.
In additional, I think that `background-position: center` is better than `background-position: 50%`.

Reply:
Okay I will fix it